pkgs/servers/misc/navidrome/default.nix

```diff
@@ -1,48 +1,40 @@
-{ lib, stdenv, fetchurl, ffmpeg, ffmpegSupport ? true, makeWrapper, nixosTests }:
+{ buildGoModule, fetchFromGitHub, lib, pkg-config, stdenv, ffmpeg, taglib, zlib, ffmpegSupport ? true, makeWrapper, nixosTests }:
 
 with lib;
 
-stdenv.mkDerivation rec {
-  pname = "navidrome";
-  version = "0.47.5";
+buildGoModule {
 
+  pname = "navidrome";
 
-  src = fetchurl (if stdenv.hostPlatform.system == "x86_64-linux"
-  then {
-    url = "https://github.com/deluan/navidrome/releases/download/v${version}/navidrome_${version}_Linux_x86_64.tar.gz";
-    sha256 = "sha256-AkSjtln53HDdIcQgnA8Wj010RXnOlOsFm2wfVgbvwtc=";
-  }
-  else {
-    url = "https://github.com/deluan/navidrome/releases/download/v${version}/navidrome_${version}_Linux_arm64.tar.gz";
-    sha256 = "sha256-+VBRiV2zKa6PwamWj/jmE4iuoohAD6oeGnlFi4/01HM=";
-  });
+  version = "0.47.5";
 
-  nativeBuildInputs = [ makeWrapper ];
+  src = fetchFromGitHub {
+    owner = "navidrome";
+    repo = "navidrome";
+    rev = "v${version}";
```

Comment:
> trying https://github.com/navidrome/navidrome/archive/v22.11pre-git.tar.gz

You're referencing the wrong version, from the wrong scope.

Reply:
Changed the top-level attrSet to `rec` to pick up the correct version